### PR TITLE
New nereon

### PR DESCRIPTION
**Pull request by drystone**

### CODE REVIEW

**Comment by strogonoff** [comment]
@drystone if there were any significant features/fixes here, could you summarize in few words? I’ll make sure to update the site

**Comment by drystone** [comment]
@strogonoff, nereond is still a placeholder for something more functional in the future. The changes in this commit were simply to bring it in line with changes to rust-nereon.

**Comment by strogonoff** [comment]
understood
